__tests__/server-get.test.js

```diff
@@ -247,3 +247,88 @@ describe("GET/api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("/api/reviews/:review_id/comments POST tests", () => {
+  test("Status: 202, Test that a new comment is posted", () => {
+    const data = {
```

Comment:
These should be 201. 

202 is just accepted and doesn't indicate anything has actually been created :)